src/v3/webpack.release.config.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { resolve } from 'path';
+import { readFileSync } from 'fs';
+
+import webpack from 'webpack';
+import { merge } from 'webpack-merge';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+import TerserPlugin from 'terser-webpack-plugin';
+import { BundleAnalyzerPlugin } from 'webpack-bundle-analyzer';
+
+import terserOptions from '../../scripts/buildtools/terser/config';
+import baseConfig from './webpack.common.config';
+
+import type { Configuration } from 'webpack';
+
+const prodConfig: Configuration = merge(
+  baseConfig,
+  {
+    mode: 'production',
+    devtool: 'source-map',
+    entry: {
+      default: {
+        import: [
+          // polyfill must appear first in entry point array
+          resolve(__dirname, '../..', './polyfill/modern.js'),
+          resolve(__dirname, 'src/index.ts'),
+        ],
+        filename: 'js/okta-sign-in.min.js',
+        library: {
+          name: 'OktaSignIn',
+          type: 'umd',
+          export: 'default',
+        },
+      },
+      noPolyfill: {
```

Comment:
maybe also add a polyfill entry? just to provide some flexibility for devs

Reply:
oh yeah, I have a TODO item that may need to go in another ticket to generate the polyfill for the g3 widget. Right now we load both entrypoints for the "full bundle" (with polyfill AND widget code) which is great, but since it's using `@babel/preset-env`, the resulting polyfill included in the bundle changes depending on the code within the bundle.

I'm not sure yet how to get webpack to also code split that polyfill into a separate file during the build process but I'm assuming it is possible with some careful code splitting plugin configuration.